tests/unit/Tests.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+import cv2
+import os
+import shutil
+import numpy as np
+import coverage
+from flow2ml import Flow
+
+# create mock data for 5 classes with 5 images each
+def create_data():
+	if os.path.exists('./dataset_dir/'):
+		shutil.rmtree('./dataset_dir/')
+	for i in range(5):
+		os.makedirs(f'./dataset_dir/data_dir/class{i}')
+		for j in range(5):
+			img = np.random.randint(0, 256, size = [150, 150, 3], dtype = np.uint8)
+			cv2.imwrite(f'./dataset_dir/data_dir/class{i}/class{i}_{j}.png', img)
+
+class testFlow2ML(unittest.TestCase):
+
+	create_data()
+	flow = Flow('dataset_dir', 'data_dir')
+	filters = ["median", "gaussian", "laplacian", "sobelx", "sobely"]
+	flow.applyFilters(filters)
```

Comment:
you should mock functions here and expect an output ratehr than calling direct function